modules/EvseManager/CMakeLists.txt

```diff
@@ -57,4 +57,9 @@ target_sources(${MODULE_NAME}
 
 # ev@c55432ab-152c-45a9-9d2e-7281d50c69c3:v1
 # insert other things like install cmds etc here
+if((${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME} OR ${PROJECT_NAME}_BUILD_TESTING) AND BUILD_TESTING)
```

Comment:
This could also just check for EVEREST_CORE_BUILD_TESTING [which is set in the toplevel CMakeLists.txt](https://github.com/EVerest/everest-core/blob/aed024ba594d9842dacbc25355407c03ba7bffa7/CMakeLists.txt#L46)